lib/cropduster.js

```diff
@@ -29,6 +29,22 @@ window.CD = {
     });
   },
 
+  throwError: function(msg) {
+    if(typeof(MICapture) != "undefined") {
```

Comment:
Should this be consistent with `suspend`, `capture`, etc to the point of checking for `typeof(MICapture) == "undefined"` rather than the inverse and using `CD.log` in favor of `console.error`?